namespaces/live-1.cloud-platform.service.justice.gov.uk/cica-oas-basic-testing/resources/ecr.tf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "cica_ecr_credentials" {
+  source     = "https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service.git"
```

Comment:
This source should be the github URL of the terraform module, not your application code.

Have a look at this one, for comparison:
https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/family-mediators-api-staging/resources/ecr.tf